### PR TITLE
Feature/refactor round one million

### PR DESCRIPTION
**Pull request by EllieJellyBean**
## Is this a fix or a feature?
Fix

## What is the change?
CSS

## What does it fix?
CSS format

## Where should the reviewer start?
aside and body input wrap

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,9 +16,9 @@ <h2>IdeaBox</h2>
   <section class="right-side">
     <div class="wrapper">
       <form class="custom-form">
-        <label for="titleInput">Title</label>
+        <label class="form-label" for="titleInput">Title</label>
         <input class="user-input title-input" id="titleInput" aria-required="true"></input>
-        <label for="bodyInput">Body</label>
+        <label class="form-label" for="bodyInput">Body</label>
```
Comment (Patfindley) [inline index.html:21]: this is smart!

File: index.html [hunk 2]
```diff
@@ -29,21 +29,7 @@ <h2>IdeaBox</h2>
     </div>
     <section class="idea-box">
       <article class="idea-grid">
-      <div class="custom-card hidden">
```
Comment (Patfindley) [inline index.html:32]: who needs it!

File: index.html [hunk 3]
```diff
@@ -29,8 +29,8 @@ <h2 class="aside-title">IdeaBox</h2>
     </div>
     <section class="idea-box">
       <article class="idea-grid">
-      <div class="custom-card hidden"></div>
```
Comment (Patfindley) [inline index.html]: nice!

File: styles.css [hunk 4]
```diff
@@ -1,7 +1,7 @@
 /* ****** GENERAL STYLES ******** */
 
 html {
-  background-color: #A9AAD2;
```
Comment (Patfindley) [inline styles.css:3]: good idea on this color choice!

File: styles.css [hunk 5]
```diff
@@ -179,33 +184,34 @@ body {
...
-  width: 18%;
+  width: 13%;
   display: inline;
 }
 
 .remove-button {
   background: url(assets/delete.svg);
   background-repeat: no-repeat;
   border: none;
-  width: 18%;
```
Comment (Patfindley) [inline styles.css:185]: nice fix!

File: index.html [hunk 6]
```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <h1><title>IdeaBox</title></h1>
+  <title class="project-title">IdeaBox</title>
```
Comment (KyleWong2510) [inline index.html:4]: Good catch

File: styles.css [hunk 7]
```diff
@@ -1,32 +1,77 @@
 /* ****** GENERAL STYLES ******** */
+
 html {
-  background-color: #A9AAD2;
+  background-color: #EAEAF4;
```
Comment (KyleWong2510) [inline styles.css:4]: Not a 'NEED TO HAVE', but look into CSS variables for repeated values like colors, fonts, font sizes/weights. Makes it easier to change a color scheme without having to find each rule and manually changing, for example. Also nice for readability.

File: styles.css [hunk 8]
```diff
@@ -1,32 +1,77 @@
...
   font-family: "Open Sans", sans-serif;
   height: 100%;
 }
+
 body {
   display: flex;
```
Comment (KyleWong2510) [inline styles.css:10]: Nice to organize these rules in some way (alphabetically, all positioning stuff grouped, etc.). If there is an agreed-upon convention for this, others can look at the code and easily make edits.  Great job organizing the overall class names and ids, though.

File: styles.css [hunk 9]
```diff
@@ -1,32 +1,77 @@
 /* ****** GENERAL STYLES ******** */
+
```
Comment (KyleWong2510) [inline styles.css:2]: Think about an '*' ruleset.  For any styles that you want to apply to all elements (i.e. display: inline-block, width: 100%, padding: none).  If you have this rule at the top, then you can specify when you want these rules broken.  I typically use something like so: 
`* {
  box-sizing: border-box;
  color: var(--secondary-text);
  font-family: var(--body-font);
  font-size: 1.25rem;
  margin: 0;
  padding: 0;
  width: 100%;
}`

**Review by KyleWong2510 — COMMENTED** [review]
Nice job, overall.  My comments are all about readability and organization.  I had to get a little nitpicky so take and leave what works for you.

**Pull request merged**